src/main/java/event.java

```diff
@@ -0,0 +1,19 @@
+public class event extends Task {
+    protected String when;
+    public event(String description, String when){
+        super(description);
+        this.when = when;
+    }
+    @Override
+    public String toString(){
+        return "[E]" + super.toString() + " (at: " + when + ")";
+    }
+
+    @Override
+    public String getWhen(){
```

Comment:
there should be an extra whitespace after the oval brackets, and before the curly brace